docs/src/main/asciidoc/refresh.adoc

```diff
@@ -30,7 +30,7 @@ The App Configuration Spring libraries support the ability to periodically check
 
 [,properties,indent=0]
 ----
-spring.cloud.azure.appconfiguration.stores[0].monitoring.refresh-interval= 1d
+spring.cloud.azure.appconfiguration.stores[0].monitoring.refresh-interval= 30s
```

Comment:
Can we set this to a longer period, like 1 minute or 5 minutes.  We had an issue in our azure docs that customer would set a refresh interval of 30s and we saw more customers running though their request quota because they never changed this later on

Reply:
Sure, I only changed it to that, as it was the default for this configuration. I was getting questions about it.

Reply:
Ah ok, if customers were asking questions about it and it is the default, then we can also keep it, your choice. Spring customers are not always expecting the same things as the ones in the rest of our docs.